docs/3.0.0-beta.x/concepts/models.md

```diff
@@ -218,7 +218,7 @@ To improve the Developer Experience when developing or using the administration
 
 ### Exceptions
 
-- `uid` — This field type allows a `targetField` key. The value is the name of an attribute thas has `string` of `text` type.
+- `uid` — This field type allows a `targetField` key. The value is the name of an attribute that has `string` of `text` type.
```

Comment:
Can you also please include a fix to the `v3.x` part of the docs for this? Thank you

Reply:
2 were already fixed, 3rd is now on 3.x branch also